src/juliaexepath.ts

```diff
@@ -277,6 +282,10 @@ export class JuliaExecutablesFeature {
         return this.actualJuliaExePath
     }
 
+    public getUsingJuliaup() {
```

Comment:
```suggestion
    public isJuliaup() {
```
reads a bit nicer, but also doesn't matter ;)

Reply:
BUT, when one has a really reliable rename functionality, it is also a very easy change, so I renamed ;)